src/Login/components/Login.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { Image, View, Text, TouchableOpacity, TextInput } from 'react-native';
+import { PropTypes } from 'prop-types';
```

Comment:
`PropTypes` is exported as default from the `prop-types` package, so we need to say `import PropTypes from 'prop-types'`

It's the difference between saying:
```JS
export const a = 1;
export default b = 2;

// in another file:
import { a } from 'module'
import b from 'module'
```